src/style/global.scss

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
+@import './anchor';
```

Comment:
Are you not able to keep it in `../anchor/anchor` and import it from there instead of from `style` folder?

Reply:
`anchor.scss` was importing from `style` folder. `global.scss` is also importing from `style`. So it would be circular dependency.

Reply:
Could we avoid importing `style` in `anchor.scss` somehow? Maybe by importing from individual files in that folder.

Because it doesn't make sense to move a component out of the main structure.

Reply:
Is it really that bad?

Reply:
And do you have an idea how not use `themes.scss` then in `anchor`?